scrapping/main.py

```diff
@@ -0,0 +1,58 @@
+import datetime
+
+import requests
+from bs4 import BeautifulSoup
+
+from scrapping.Day import Day
+from scrapping.Processor import *
+from scrapping.Link import Link
+from django.db.models import Model
+from django.db import models
+from schedule.models import *
+
+def convertToModel(ora):
+    for grupa in ora.grupe:
```

Comment:
Aici ar fi bine pt prod ca e slabut sa faci batchuri de 50/100 sa le trimiti pe alea si sa dai un sleep pana la urmatorul. Also ai putea pune un delete all inainte sa adaugam mereu pe o baza goala.

Reply:
Am pus un sleep la fiecare zi (e comentat linia 31 in main), pentru fiecare zi din fiecare serie (in fiecare zi avem in medie la seriile mari 80-100 de ore)